src/query-builder/QueryBuilder.ts

```diff
@@ -1639,13 +1646,26 @@ export class QueryBuilder<Entity> {
         return "";
     }
 
+    createSpecificExpression(sql: string): string {
```

Comment:
also please name it better, its not understandable what this specific is. Something like `createLimitOffsetSpecificExpression`